### PR TITLE
Add update-password endpoint to Profile API

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: spec/paths/profile@update-password.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    Updates user's password with the specified newPassword. And checks if currentPassword matches the actual one.
+  parameters:
+    - name: body
+      in: body
+      description: currentPassword and newPassword
+      required: true
+      schema:
+        $ref: "#/definitions/UpdatePassword"
+  responses:
+    201:
```
Comment (vova07) [inline spec/paths/profile@update-password.yaml]: Do we create a new resource ?

Comment (XaosSintez) [inline spec/paths/profile@update-password.yaml]: Well, we response with 201 when we cancel subscription or anything else like that. And also we send 201 in `users/{id}/password` too

Comment (vova07) [inline spec/paths/profile@update-password.yaml]: hmmm ... seems incorrect.
Not sure how we miss that.

Comment (XaosSintez) [inline spec/paths/profile@update-password.yaml]: We redirect to Profile after request (as we do  in many other places), so 201 seems correct

Comment (slavcodev) [inline spec/paths/profile@update-password.yaml]: I think `201` is right. And yes we are created new resource - *new password`

**Comment by slavcodev** [comment]
`profile@update-password` -> `profile@passwords` ?

**Comment by XaosSintez** [comment]
@slavcodev there is no `passwords`, it is current user's password

**Pull request merged**